### PR TITLE
pio-*: add Dutch translation

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.nl/common/pio-lib.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`pio lib list`
+
+- Toon ingebouwde bibliotheken gebaseerd op geïnstalleerde ontwikkelplatformen en hun geraamtes:
+
+`pio lib builtin`
+
+- Zoek naar bestaande bibliotheken:
+
+`pio lib search {{keyword}}`
```
Comment (gutjuri) [inline pages.nl/common/pio-lib.md:16]: Can `keyword` also be translated?

File: pages.nl/common/pio-remote.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# pio remote
+
+> Helper commando voor PlatformIO Remote Development.
+> `pio remote [command]` accepteert dezelfde argumenten als de lokale tegenhanger `pio [command]`.
```
Comment (gutjuri) [inline pages.nl/common/pio-remote.md:4]:
```suggestion
> `pio remote [commando]` accepteert dezelfde argumenten als de lokale tegenhanger `pio [commando]`.
```

File: pages.nl/common/pio-remote.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+> Meer informatie: <https://docs.platformio.org/en/latest/core/userguide/remote/index.html>.
+
+- Toon alle actieve Remote Agents:
```
Comment (gutjuri) [inline pages.nl/common/pio-remote.md:7]: Is there an appropriate translation for `remote agent`?

File: pages.nl/common/pio-upgrade.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# pio upgrade
+
+> Update PlatformIO naar de laatste versie.
+> Meer informatie: <https://docs.platformio.org/en/latest/core/userguide/cmd_upgrade.html>.
+
+- Update PlatformIO naar de laatste versie.:
```
Comment (gutjuri) [inline pages.nl/common/pio-upgrade.md:6]:
```suggestion
- Update PlatformIO naar de laatste versie:
```

File: pages.nl/common/pio.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# pio
+
+> Ontwikkelomgeving voor voor embedded boards.
+> Sommige subcommando's zoals `pio run` hebben hun eigen documentatie.
+> Meer informatie: <https://docs.platformio.org/en/latest/core/userguide/>.
+
+- Toon help en toon subcommando's:
```
Comment (gutjuri) [inline pages.nl/common/pio.md:7]:
```suggestion
- Toon help en toon subcommandos:
```

If apostrophies are used in Dutch as they are in English and German, this should be changed. However, please let me know if I'm wrong on  this one.

**Review by gutjuri — COMMENTED** [review]
Thanks for this massive PR! I left some suggestions.

File: pages.nl/common/pio-package.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+- Verwijder alle versies van een gepubliceerd pakket van het register:
+
+`pio package unpublish {{pakket}}`
+
+- Verwijder een specifieke versie van een gepubliceerd pakket van het register:
+
+`pio package unpublish {{pakket}}@{{version}}`
+
+- Maak de verwijdering ongedaan en zet alle versies of een sepcifieke versie van het pakket terug in het register:
```
Comment (leonvsc) [inline pages.nl/common/pio-package.md:35]:
```suggestion
- Maak de verwijdering ongedaan en zet alle versies of een specifieke versie van het pakket terug in het register:
```

File: pages.nl/common/pio-remote.md (hunk 3 above)
Comment (leonvsc) [reply pages.nl/common/pio-remote.md:7]: I think the same as @sebastiaanspeck. The best idea is to keep `remote agent`. It is a pretty standard term in the IT in The Netherlands.

File: pages.nl/common/pio-run.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`pio run`
+
+- Voer alle doelen uit voor de gespecificeerde omgevingen:
+
+`pio run --environment {{omgeving1}} --environment {{omgeving2}}`
+
+- Voer gespecificeerde doelen uit:
+
+`pio run --target {{target1}} --target {{target2}}`
```
Comment (leonvsc) [inline pages.nl/common/pio-run.md:24]:
```suggestion
`pio run --target {{doel1}} --target {{doel2}}`
```

File: pages.nl/common/pio-system.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+# pio system
+
+> Gemengde systeem commando's for PlatformIO.
```
Comment (leonvsc) [inline pages.nl/common/pio-system.md:3]:
```suggestion
> Gemengde systeem commando's voor PlatformIO.
```

File: pages.nl/common/pio-test.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+- Test alleen op specifieke omgevingen:
+
+`pio test --environment {{omgeving1}} --environment {{omgeving2}}`
+
+- Voer alleen testen uit die qua naam overeenkomen met een specifiek glob patroon:
+
+`pio test --filter "{{patroon}}"`
+
+- Negeer testen die qua naam name overeenkomen met een specifiek glob patroon:
```
Comment (leonvsc) [inline pages.nl/common/pio-test.md:18]:
```suggestion
- Negeer testen die qua naam overeenkomen met een specifiek glob patroon:
```

File: pages.nl/common/pio-test.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+`pio test --ignore "{{patroon}}"`
+
+- Specificeer een port voor firmware uploading:
```
Comment (leonvsc) [inline pages.nl/common/pio-test.md:22]:
```suggestion
- Specificeer een poort voor firmware uploading
```

**Review by leonvsc — APPROVED** [review]
Thanks for this massive PR! It LGTM after the suggestions!

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**